CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ option(ZAP_TEST
        "ZAP fuzz test using https://www.zaproxy.org/docs/docker/api-scan/" OFF
 )
 option(BUILD_SMALLBANK "Build SmallBank sample app and clients" ON)
+option(BUILD_TPCC "Build tppc sample app and clients" ON)
 
 # Build common library for CCF enclaves
```

Comment:
So we are building smallbank by default, and we have to have a -DBUILD_SMALLBANK=OFF in our CI Debug config (matrix.yml) as a result. We can either do the same for TPCC, or default both to OFF and add a flag to enable them in the Perf job. I think I mildly prefer the latter, but so long as we do one of those I think I'm happy.